src/clearwater_modules/nsm1/constants.py

```diff
@@ -356,21 +284,24 @@ class GlobalVars(TypedDict):
     kaw_20_user = 999,
     kah_20_user = 999,
     hydraulic_reaeration_option = 2,
-    wind_reaeration_option = 2,
-    Fr_PAR = .5,
-    lambda0 = .5,
-    lambda1 = .5,
-    lambda2 = .5,
-    lambdas = .5,
-    lambdam = .5,    
-    timestep = 86400,
+    wind_reaeration_option = 2,  
+    timestep = 1,    #TODO Dynamic or static?
+    depth = 1.5,         #TODO Dynamic or static?
```

Comment:
can remove these to dos?

Reply:
If this timestep refers to `dt`, recommend changing the name.

Reply:
"compute_depth" is a process under the shared\processes.py based on surface_area and volume but depth is defined as a static variable, I was unsure if this was supposed to be dynamic or static. I am still a little confused between timestep and dt.

Reply:
Ah, gotcha. That makes sense for TSM, where `surface_area` and `volume` are both state variables, so in that context, depth was a dynamic variable. I'm not as familiar with NSM, but if those aren't state variables for NSM (which it appears they are not), then depth would either need to be a `static` or `state` variable for NSM. It is interesting that none of surface area, volume, depth are state variables... @aufdenkampe, thoughts?

With recent updates made in #77, a `Model` instance is now instantiated to have a `timestep`, which tracks which number timestep we are on (e.g., start at `timestep=0`, then increment to `timestep=1`, then increment to `timestep=2` and so on). This isn't used in any of the processes, but is rather used to help index the xarray. This makes the use of a `timestep` variable within the processes a bit more confusing, even though it still works. In the context of the processes, the `timestep` variable is really referring to the amount of time that has passed within the timestep, which is why I propose to rename to `dt`. However, let's leave this for now so we can get this PR merged -- we can address as part of #79

Reply:
@kewalak & @sjordan29, We want to handle a grid cell water `volume`, `depth`, and wetted `surface_area` identically in all modules in this repo. All three can change during a model run, so none of them should be considered `static`. The question is whether we read them all as inputs or calculate any of them.  @xaviernogueira brought up these questions in:
- #50 

In that thread I describe how our use of the terms `static`, `state`, and `dynamic` is quite problematic, and I created the following issue to eventually move toward a better approach:
- #65 

For now, I think it's reasonable to call them all `dynamic` or `state`, but definitely not `static`. In the future these will have an `intent` of `in` or `inout`.

Reply:
If we want to handle identically to TSM, we would want to have `surface_area` and `volume` be state variables, making `depth` a dynamic variable calculated from those two state variables.

However, in this case, it seems like handling identically to TSM does not necessarily make sense: unlike TSM, `surface_area` and `volume` are not used in processes (besides the shared process referenced above which computes depth). Only `depth` is used in the processes for nsm as far as I can tell, so we'd be requesting more information than is needed. 

But I see your point in the instance where you're running both TSM and NSM. You'd provide `surface_area` and `volume` for TSM, and you'd want to use the same calculated `depth` for both TSM and NSM calculations.

Reply:
To be clear, variables are not defined in one location - each module has its own variables defined, but there's also a "shared" module with shared processes, types, and variables. That does provide module-specific flexibility (i.e., we could make depth a state variable for NSM and dynamic for another, though we'd want to think about the implications of that when running multiple modules). Looking closer, it doesn't look like TSM uses depth at all. 

This seems like a bigger question than this specific PR -- feeling like we should get this merged in so that the testing is in place. Shall we move this discussion to a few issues? Thinking one specifically on how to handle this case, and perhaps another on developing a vision for running multiple modules at once. I think it would be helpful to discuss in person in our check in tomorrow as well.

Reply:
@sjordan29, I like your suggestion to move forward with this PR, and spin off a separate issue to discuss the case where we might run multiple modules at a time. Thanks!

Reply:
Created #83 and #84 which we can build on later.